css/scss/sections/_index.scss

```diff
@@ -0,0 +1,228 @@
+body {
+  background-color: #307367;
+  @include breakpoint(medium) {
+  background-color: #fff;
+  }
+}
+
+
+p{
+  font-weight: bold;
+color:#fff;
+  @include breakpoint(medium) {
+ color: set-color('grey-700');
+  }
+}
+
+.loginCon {
+ padding: 0;
+
+
+
+    .row {
+    padding:0;
+    }  
+}
+
+.left-log  {
+  padding: 0;
+  margin: 0;
+.bgimg {
+    background-image: url('#{$img-path}/cto-bg2.jpg');
+    height: 100%;
+    min-height: 100vh;
+    background-size: cover;
+}
+}
+
+.right-log {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around; 
+  width: 70%; 
+   @include breakpoint(medium) {
+  justify-content: space-around; 
+  width: 57%; 
+  }
+   @include breakpoint(large) {
+  justify-content: flex-end; 
+  width: 41%;     
+  }
+}
+
+.whiteLogo {
+  margin-top: 80px;
+  width:150px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+
+.vertical-logo img {
+  margin-top: 80px;
+  width:100px; 
+  @include breakpoint(large) {
+  width:150px;       
+  }
+}
+
+.checkbox {
+  font-weight: normal;
+  margin-top: auto;
+  margin-bottom: auto;
+  @include breakpoint(medium) {
+ color: set-color('grey-700');
+     }
+}  
+
+.link {
+  font-weight: normal;
+  margin-top: auto;
+  margin-bottom: auto;
+   color: #fff;
+  @include breakpoint(medium) {
+ color: set-color('grey-700');
+     }
+ }
+.register {
+  .column-small-6{
```

Comment:
Why are you overriding the global column styling? if this is not the correct column width you need, you should use another class (column-small-4 or column-small-5, etc..), and not change the existing class.

Reply:
fixed